actions/kustomize-gke-deploy/action.yaml

```diff
@@ -0,0 +1,18 @@
+name: "Deploy kustomized crds and app"
+description: "Deploy kustomized crds and app to the GKE cluster"
+inputs:
+  kustomization-path:
+    description: "Path to the root directory of the kustomization"
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Deploy kustomized crds
+      shell: bash
+      run: |
+        kubectl kustomize ${{ inputs.kustomization-path }} | kfilt -i kind=CustomResourceDefinition | kubectl apply -f -
+        kubectl kustomize ${{ inputs.kustomization-path }} | kfilt -i kind=CustomResourceDefinition | kubectl wait --for condition=established --timeout=60s -f -
```

Comment:
Could you make the timeout configurable through an input? That would be very convenient in case a cluster is taking longer in the future to deploy CRDs